store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -1537,27 +1545,35 @@ private boolean isSpaceToDelete() {
             cleanImmediately = false;
 
             {
-                String storePathPhysic = DefaultMessageStore.this.getMessageStoreConfig().getStorePathCommitLog();
-                double physicRatio = UtilAll.getDiskPartitionSpaceUsedPercent(storePathPhysic);
-                if (physicRatio > diskSpaceWarningLevelRatio) {
-                    boolean diskok = DefaultMessageStore.this.runningFlags.getAndMakeDiskFull();
-                    if (diskok) {
-                        DefaultMessageStore.log.error("physic disk maybe full soon " + physicRatio + ", so mark disk full");
-                    }
-
-                    cleanImmediately = true;
-                } else if (physicRatio > diskSpaceCleanForciblyRatio) {
-                    cleanImmediately = true;
+                List<String> storePaths;
+                if (DefaultMessageStore.this.getMessageStoreConfig().isMultiCommitLogPathEnable()) {
+                    storePaths = DefaultMessageStore.this.getMessageStoreConfig().getCommitLogStorePaths();
                 } else {
-                    boolean diskok = DefaultMessageStore.this.runningFlags.getAndMakeDiskOK();
-                    if (!diskok) {
-                        DefaultMessageStore.log.info("physic disk space OK " + physicRatio + ", so mark disk ok");
-                    }
+                    storePaths = Collections.singletonList(DefaultMessageStore.this.getMessageStoreConfig().getStorePathCommitLog());
                 }
 
-                if (physicRatio < 0 || physicRatio > ratio) {
-                    DefaultMessageStore.log.info("physic disk maybe full soon, so reclaim space, " + physicRatio);
-                    return true;
+                for (String storePathPhysic : storePaths) {
+                    double physicRatio = UtilAll.getDiskPartitionSpaceUsedPercent(storePathPhysic);
+                    if (physicRatio > diskSpaceWarningLevelRatio) {
```

Comment:
If all storePath is full, then treat it as full.
To co-operate with this, the create method should consider the capacity of storePath.

Reply:
If the storePath is marked to be unmounted, just ignore it.